src/ui/actions/app.ts

```diff
@@ -375,3 +377,21 @@ export function setShowVideoPanel(showVideoPanel: boolean): SetShowVideoPanelAct
 export function setShowEditor(showEditor: boolean): SetShowEditorAction {
   return { type: "set_show_editor", showEditor };
 }
+
+export function loadReplayPrefs(recordingId: RecordingId): UIThunkAction {
+  return async ({ dispatch }) => {
+    const replaySessions = await asyncStore.replaySessions;
+    const session = replaySessions[recordingId];
+
+    if (!session) {
```

Comment:
I assume this represents the case where the user has never viewed that recording before. In that case, should we load the default settings? Having a `DEFAULT_SETTINGS` const would also be a nice way of centralizing those values. I know they also live in prefs, sort of, but pulling out a well-named constant might be more self-documenting and central.

Reply:
#4047.